tensorboard/webapp/metrics/store/metrics_reducers_test.ts

```diff
@@ -2007,7 +2007,7 @@ describe('metrics reducers', () => {
         });
       });
 
-      it('sets step index to null if time series is empty', () => {
+      it('does not reset step index to null when time series is empty', () => {
```

Comment:
I think you can just delete the test. Pretend the logic to do this never existed and we never thought there was a need to write a test for it.